src/main/java/io/temporal/internal/sync/SyncActivityWorker.java

```diff
@@ -39,11 +39,14 @@ public SyncActivityWorker(
       WorkflowServiceStubs service,
       String namespace,
       String taskList,
+      double taskListActivitiesPerSecond,
```

Comment:
Why pass this separately and not use ```SingleWorkerOptions``` value for ```taskListActivitiesPerSecond```? https://github.com/temporalio/temporal-java-sdk/blob/master/src/main/java/io/temporal/internal/worker/SingleWorkerOptions.java#L45

Reply:
Good catch. Removed taskListActivitiesPerSecond from SinglePollerOptions as they don't apply to the workflow worker.